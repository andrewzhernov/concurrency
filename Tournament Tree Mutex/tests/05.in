3 330000 1
