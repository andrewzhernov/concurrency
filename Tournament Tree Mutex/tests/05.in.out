Testing with parameters: n_threads=3, n_iters=330000, n_shareds=1
Starting 3 threads ... 
Done
Joining 3 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=3, n_iters=330000, n_shareds=1
Starting 3 threads ... 
Done
Joining 3 threads ... 
Done
Checking test invariant ... 
Done
OK
Testing with parameters: n_threads=3, n_iters=330000, n_shareds=1
Starting 3 threads ... 
Done
Joining 3 threads ... 
Done
Checking test invariant ... 
Done
OK
